sound/hybrid_font.h

```diff
@@ -527,13 +527,14 @@ class HybridFont : public SaberBase, public Looper {
 	if (num_alternatives) {
 	  if (SaberBase::sound_number == -1) {
 	    // Next alternative
-	    if (++current_alternative >= num_alternatives)  current_alternative = 0;
+	    ++current_alternative;
 	  } else {
 	    // Select a specific alternative.
 	    current_alternative = std::min<int>(SaberBase::sound_number, num_alternatives);
 	    // Set the sound num to -1 so that the altchng sound is random.
 	    SaberBase::sound_number = -1;
 	  }
+          if (current_alternative >= num_alternatives) current_alternative = 0;
```

Comment:
I thought you didn't want wrap-around for alternatives?

Reply:
need to think this through, need to figure out best approach so the style can't set an alt that doesn't exist because all sounds in the alt stop when that happens.  will try out a few things tomorrow and update